### DISCUSSION TITLE
Performance Issues with BarHitErrorMeter

### DISCUSSION THREAD

**Discussion by letsgoawaydev** [ppy/osu#23310] [Q&A] [answered]
I'll make a proper issue later, but ive noticed that on mobile removing barhiterror improves performance significantly.
https://youtu.be/SDfJrzqhvNQ
here are the settings i used in the video
![F02C5D6C-6E8B-4A33-879B-E2E8C8B2DB77](https://user-images.githubusercontent.com/68365423/234356037-8ae56018-0e21-452b-ab91-e6c30375a30c.jpeg)
iPad 8th Generation iPadOS16.1

**Comment by letsgoawaydev** [comment]
also ill add, hardware accel being on doesnt change things, fps still drops

**Reply by peppy** [reply]
Hardware acceleration is for video decoding. It does not apply here.

**Comment by peppy** [comment, marked as answer]
This isn't going to be anything specific to the bar error metre, that's probably just pushing things over the edge. We're constantly working on performance so stay tuned for future optimisations.
